test/v25_net_test.py

```diff
@@ -6,55 +6,66 @@
 
 from test_base import TestBase
 
-graph_file_path = Path(os.path.dirname(__file__)) / "data" / "12_node_ring.graphml"
 
-base = TestBase()
-base.start_server()
-print(base.warcli(f"network start {graph_file_path}"))
-base.wait_for_all_tanks_status(target="running")
-base.wait_for_all_edges()
-
-onion_addr = None
-
-
-def wait_for_reachability():
-    try:
-        global onion_addr
-        info = json.loads(base.warcli("rpc 0 getnetworkinfo"))
-        for net in info["networks"]:
-            if net["name"] == "ipv4" and not net["reachable"]:
+class V25NetTest(TestBase):
```

Comment:
0f439f1a5812cd033516fccb56d14b1ea7e4a103

lets rename this test class and file `OnionTest`, we haven't used bitcoin core v25 in a year :-/ see #306 

I appreciate you cleaning this one up even though we cant use it yet, thanks.

Reply:
nice, done